src/NUnit.TestAdapter.Tests.Acceptance/NativeMethods.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace NUnit.VisualStudio.TestAdapter.Tests.Acceptance
+{
+    internal static class NativeMethods
+    {
+        /// <summary>
+        /// https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createdirectoryw
+        /// </summary>
+        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool CreateDirectory(string lpPathName, IntPtr lpSecurityAttributes);
```

Comment:
We don't want to require Windows though. I'll use a locked file in the directory to guarantee mutual exclusivity.